components/pedigree/ui/src/main/resources/PhenoTips/UIX_Field__pedigree.xml

```diff
@@ -720,7 +720,7 @@ $xwiki.jsx.use('PhenoTips.FamilySelector')##
     #end
     (% class="pedigree-wrapper" %)((({{html clean="false"}}
     &lt;input type="hidden" id="familyIdentifier" value="$!{family.getId()}" /&gt;
-    &lt;div style="display:inline-block;" id="pedigree-thumbnail" class="pedigree-box ${pClass} #if($canEditP) pedigree-edit-trigger #end" title="${pAction}" &gt;${pContent}&lt;/div&gt;
+    &lt;div style="display:inline-block; #if(!$canEditP)cursor:auto;#end" id="pedigree-thumbnail" class="pedigree-box ${pClass} #if($canEditP) pedigree-edit-trigger #end" title="${pAction}" &gt;${pContent}&lt;/div&gt;
```

Comment:
By some reason there is no notification if you edit your comment, so I just saw it, but I still did same change :)